bitshares_pricefeed/pricefeed.py

```diff
@@ -98,31 +98,35 @@ def obtain_flags(self, symbol):
 
         # Feed too old
         feed_age = self.price_result[symbol]["current_feed"]["date"] if self.price_result[symbol]["current_feed"] else datetime.min
-        if (datetime.utcnow() - feed_age).total_seconds() > self.assetconf(symbol, "maxage"):
+        if (datetime.now(timezone.utc) - feed_age).total_seconds() > self.assetconf(symbol, "maxage"):
```

Comment:
why that change?

Reply:
See issue #57  and [this commit](https://github.com/Zapata/bitshares-pricefeed/commit/066b4c61972203b739fc4ba81bd46428abb4a7ea)

Reply:
See issue #57  and [this commit](https://github.com/Zapata/bitshares-pricefeed/commit/066b4c61972203b739fc4ba81bd46428abb4a7ea)

Reply:
More specifically utcnow() [returns a date with timezone=None](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow), and `current_feed.date` is, since the last python-bitshares upgrade, with a timezone.